templates/influxdb.conf.epp

```diff
@@ -12,15 +12,18 @@
       Array $master_list
```

Comment:
Make this `Array[String]` to avoid having to deal with bad stuff in the `$master_list` (later there are calls to `split` which will fail if there is garbage in `$master_list`